gobblin-runtime/src/main/java/gobblin/runtime/util/JobMetrics.java

```diff
@@ -87,6 +88,9 @@ private static String name(JobState jobState) {
     List<Tag<?>> tags = Lists.newArrayList();
     tags.add(new Tag<String>("jobName", jobState.getJobName() == null ? "" : jobState.getJobName()));
     tags.add(new Tag<String>("jobId", jobState.getJobId()));
+    if (jobState.contains(ConfigurationKeys.HADOOP_CLUSTER_NAME_KEY)) {
```

Comment:
It will be great if this can be automatically inferred, instead of relying on yet another config. Say, use the host name of the Name Node.